lama.lua

```diff
@@ -84,6 +86,20 @@ facing.load = function() --loads facing / current movement direction
 	end
 end
 
+equipment.save = function()
```

Comment:
This won't work because you haven't defined a table as equipment.  Do that instead of left/right